.github/workflows/main.yml

```
@@ -15,13 +15,10 @@ jobs:
           node-version: "14"
 
       - name: Install
-        run: npm install -g vsce && npm install
+        run: yarn
```

To avoid changes in `yarn.lock` on CI:

```suggestion
        run: yarn install --frozen-lockfile
```